staging/src/k8s.io/apiextensions-apiserver/test/integration/ratcheting_test.go

```diff
@@ -174,24 +180,31 @@ func (a applyPatchOperation) Do(ctx *ratchetingTestContext) error {
 		return fmt.Errorf("invalid patch type: %T", a.patch)
 	}
 
+	if ctx.StatusSubresource {
+		patch.Object = map[string]interface{}{"status": patch.Object}
+	}
+
 	patch.SetKind(kind)
 	patch.SetAPIVersion(a.gvr.GroupVersion().String())
 	patch.SetName(a.name)
 	patch.SetNamespace("default")
 
-	_, err := ctx.DynamicClient.
-		Resource(a.gvr).
-		Namespace(patch.GetNamespace()).
-		Apply(
-			context.TODO(),
-			patch.GetName(),
-			patch,
-			metav1.ApplyOptions{
-				FieldManager: "manager",
-			})
+	c := ctx.DynamicClient.Resource(a.gvr).Namespace(patch.GetNamespace())
+	if ctx.StatusSubresource {
+		if _, err := c.Get(context.TODO(), patch.GetName(), metav1.GetOptions{}); apierrors.IsNotFound(err) {
+			// ApplyStatus will not automatically create an object, we must make sure it exists before we can
```

Comment:
nit: any deeper reason why we use Apply in the first place for creations? A `createOperation` might be clearer.

Reply:
It's how I found it to be honest.

For the non subresource path, if you apply an object it auto-creates the object if it doesn't exist, or updates it if it does. So I guess that was cleaner since this operation is used throughout tests not only to create objects, but also update them.

Would you prefer to see a `createOperation` that is used in all tests to make sure the object exists? I guess it would create the object with an empty spec/status and then all apply operations would definitely be updates, not sometimes creates